Ginger/GingerCore/Drivers/JavaDriverLib/JavaDriver.cs

```diff
@@ -2402,7 +2402,11 @@ private List<ElementInfo> GetElementsFromPL(PayLoad PLRC)
             return list;
         }
 
-        public override void StartRecording()
+        public event Amdocs.Ginger.Plugin.Core.ElementRecordedEventHandler ElementRecorded;
```

Comment:
do not replace the  public override void StartRecording() functions so old recorsding mechnisem still will work, add the void Amdocs.Ginger.Plugin.Core.IRecord.StartRecording() on top and in both call to the same recording handling code.

please do the same for StopRecording and to all other drivers like: PB, Appium, ...

Reply:
As disscussed, tested but the "StartRecording" method doesnt come from the "IWindowExplorer", it comes from the "DriverBase" so currently implemented both methods the "public override void StartRecording" and "void Amdocs.Ginger.Plugin.Core.IRecord.StartRecording()", later we will remove the "override" method from drivers